ivy/functional/ivy/activations.py

```diff
@@ -557,6 +593,9 @@ def softplus(
     return current_backend(x).softplus(x, beta=beta, threshold=threshold, out=out)
 
 
+softplus.jax_like = _softplus_jax_like
```

Comment:
Since softplus is well defined on the complex numbers, I'd suggest that unless you're including `threshold` it would be better handled with changes to the backend functions rather than with a `jax_like` function here